projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-full-access-view/case-full-access-view.component.html

```diff
@@ -59,14 +59,14 @@ <h2 class="heading-h2 error-summary-heading" id="edit-case-event_error-summary-h
                       <th id="case-viewer-field-label" *ngIf="!isFieldToHaveNoLabel(field)">
                         <div class="case-viewer-label text-16">{{field.label}}</div>
                       </th>
-                      <th>
+                      <td>
```

Comment:
@johnbenjamin-hmcts - the `<td>` is back again...?

Reply:
yes @Daniel-Lam, as we discussed earlier it should be `td` and not `th` as per the ticket https://tools.hmcts.net/jira/browse/EUI-6551 you mentioned during our chat on 11th November in team-x channel.